importers/djornl/parser.py

```diff
@@ -276,11 +272,16 @@ def process_file(self, file, remap_fn, store_fn, err_list, validator=None):
         """
         print("Parsing " + file["data_type"] + " file " + file["file_path"])
         file_parser = self.parser_gen(file)
+
+        def add_error(error):
+            print(error)
+            err_list.append(error)
```

Comment:
this will result in errors getting printed out twice, since they're already printed in the summary section

Reply:
Yes, this is a vestige of the other code. I have removed the print in the commit below.